app/services/catalog_remediator.rb

```diff
@@ -49,20 +53,21 @@ def zipped_moab_versions_beyond_expiry
 
   def zipped_moab_versions_with_errors
     zipped_moab_versions_beyond_expiry.to_a.filter_map do |zipped_moab_version|
-      audit_results = audit_results_for(zipped_moab_version)
-      # If this check returns false, it indicates a lack of zip parts, in which
-      # case we still want to delete the ZMV, so short-circuit the rest of the
-      # block
+      audit_results = empty_audit_results(zipped_moab_version)
+      # when Audit::CatalogToArchive.check_child_zip_part_attributes() returns true, we have all ZipParts for the
+      #   ZippedMoabVersion, so continue to the next check
       next [zipped_moab_version, audit_results] unless check_child_zip_part_attributes(zipped_moab_version, audit_results)
 
+      # S3Audit.check_replicated_zipped_moab_version creates audit_results.error_results if problem with existence or
+      #   checksum of replicated files
       endpoint_audit_class_for(zipped_moab_version).check_replicated_zipped_moab_version(zipped_moab_version, audit_results, true)
       next if audit_results.error_results.empty?
 
       [zipped_moab_version, audit_results]
     end
   end
 
-  def audit_results_for(zipped_moab_version)
+  def empty_audit_results(zipped_moab_version)
```

Comment:
Are they necessarily empty? I'm not sure I get this change.

Reply:
they're empty until the checks are done, no?  thats the impression I got from John when we were speaking earlier.

Reply:
the method that's being renamed does, i believe, return an empty data structure, which is then populated by the subsequent checks its passed to.  i don't know that either of these names is as clear as i'd like, but i also don't have a better suggestion.

Reply:
I just double-checked, and it is empty.